commit 8ace358d65059152d9a1f43f4770170d29d35754
Author: Jia Tan <jiat0218@gmail.com>
Date:   Fri Dec 16 20:58:55 2022 +0800

    CMake: Update .gitignore for CMake artifacts from in source build.
    
    In source builds are not recommended, but we can make it easier
    by ignoring the generated artifacts from CMake.

diff --git a/.gitignore b/.gitignore
index ead246c7..2804cfd9 100644
--- a/.gitignore
+++ b/.gitignore
@@ -5,6 +5,7 @@
 
 .deps
 .libs
+*.a
 *.gcda
 *.gcno
 *.la
@@ -77,3 +78,25 @@ coverage
 /src/liblzma/api/Makefile
 /src/lzmainfo/Makefile
 /src/xzdec/Makefile
+
+/CMakeCache.txt
+/CMakeFiles
+/CTestTestfile.cmake
+/cmake_install.cmake
+/DartConfiguration.tcl
+/liblzma-config-version.cmake
+/liblzma-config.cmake
+/lzcat
+/lzcat.1
+/lzma
+/lzma.1
+/Testing
+/tests_bin/
+/unlzma
+/unlzma.1
+/unxz
+/unxz.1
+/xz
+/xzcat
+/xzcat.1
+/xzdec